sklearn/decomposition/incremental_pca.py

```diff
@@ -21,11 +21,12 @@ class IncrementalPCA(_BasePCA):
     but not scaled for each feature before applying the SVD.
 
     Depending on the size of the input data, this algorithm can be much more
-    memory efficient than a PCA.
+    memory efficient than a PCA, and allows sparse input.
 
     This algorithm has constant memory complexity, on the order
-    of ``batch_size``, enabling use of np.memmap files without loading the
-    entire file into memory.
+    of ``batch_size * n_features``, enabling use of np.memmap files without
+    loading the entire file into memory, and the densification of sparse
+    matrices without storing the entire dense matrix at any one time.
```

Comment:
@jnothman does this resolve your comment about the computational cost of densification?